images/win/scripts/Installers/Install-LLVM.ps1

```diff
@@ -0,0 +1,7 @@
+################################################################################
+##  File:  Install-LLVM.ps1
+##  Desc:  Install LLVM
+################################################################################
+
+# Install latest LLVM in chocolatey
+Choco-Install -PackageName llvm
```

Comment:
Please update toolset.json file [common_packages - section] - https://github.com/actions/virtual-environments/blob/main/images/win/toolsets/toolset-2019.json

Reply:
Just to confirm, this is instead of my current changes ?

Reply:
Yep, no need to create a separate script to install llvm

Reply:
Added to all three toolsets.

Reply:
Anything else I need to do for this to move forward ?

Reply:
@henti, Waiting for checks to complete.

Reply:
@al-cheb Looks like tests all completed without any issues.